Packs/jamf/Integrations/jamfV2/jamfV2.py

```diff
@@ -1285,6 +1325,17 @@ def endpoint_command(client, args):
     return command_results
 
 
+def check_authentication_parameters(client_id: str | None, client_secret: str | None,
+                                    username: str | None, password: str | None) -> DemistoException | None:
+    """
+    Validate that the authentication parameters are correctly provided
+    """
+    if (not all([client_id and client_secret]) and not all([username and password])) or \
+            any([client_id, client_secret]) and any([username, password]):
+        raise DemistoException(AUTHENTICATION_PARAMS_ERROR)
+    return None
+
+
```

Comment:
If we create a client, the parameters will be validated as part of it, I dont think we need it here as well. The only pros is that you do not even try to initiate the client, its not major. WDYT?